src/apps/competitions/migrations/0008_auto_20180130_1104.py

```diff
@@ -1,7 +1,7 @@
 # Generated by Django 2.0.1 on 2018-01-30 11:04
 
 from django.db import migrations, models
-import utils.data
+import src.utils.data
```

Comment:
remove `src`

Reply:
Needed because of dockerize

Reply:
We should change this in that branch... Or change how we're doing things there

Reply:
This def shouldn't be needed, src isn't even a module so it shouldn't even work... confused on this one